vue-frontend/src/components/contact/ContactFormFooter.vue

```diff
@@ -329,9 +329,9 @@ export default {
         var head = document.getElementsByTagName("head")[0];
         var script = document.createElement("script");
         script.type = "text/javascript";
-        script.onload = () => {
+        script.onload = async () => {
           // verify recaptcha
-          grecaptcha.enterprise.ready(() => {
+          await grecaptcha.enterprise.ready(() => {
```

Comment:
Its showing an invalid recaptcha score, though I have set it as env var for both jobs and contact

Reply:
created ticket.